e2e/tests/ui-driven/src/create-flow-with-geospatial.spec.ts

```diff
@@ -61,8 +60,10 @@ test.describe("Flow creation, publish and preview", () => {
     await tearDownTestContext();
   });
 
-  test("Create a flow", async ({ browser }) => {
-    test.setTimeout(70_000);
+  test("Create, publish, and preview a flow with geospatial components", async ({
+    browser,
+  }) => {
+    test.setTimeout(140_000);
```

Comment:
This is now one large test instead of two.

Previously, the following pattern was happening -
 - Test 1 fails
 - Test 2 fails (because it depends on the flow created by test 1)
 - Test 1 re-runs, and passes, it's just flakey
 - Report shows that Test 1 passed, Test 2 failed, but without meaningful logs or errors as nothing failed per-se - the test just wasn't set up correctly.